moon/test_artifacts.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+from unittest.mock import MagicMock
+from pathlib import Path
+import numpy as np
```

Comment:
I would sort the import in 3 groups separated by empty line:
- python base libraries
- 3rd party (numpy, cv2)
- our osgar imports

Reply:
ok